gpiozero/boards.py

```diff
@@ -2239,3 +2239,104 @@ def off(self):
         """
         self.buzzer.value = None
         super(JamHat, self).off()
+
+class Pibrella(CompositeOutputDevice):
+    """
+    Extends :class:`CompositeOutputDevice` for the Cyntech/Pimoroni `Pibrella`_
+    board.
+
+    The Pibrella board comprises 3 LEDs, a button, a tonal buzzer, four general
+    purpose input channels, and four general purpose output channels (with LEDs).
+
+    This class exposes the LEDs, button and buzzer.
+
+    Usage::
+
+        from gpiozero import Pibrella
+
+        pb = Pibrella()
+
+        pb.button.wait_for_press()
+        pb.lights.on()
+        pb.buzzer.play('A4')
+        pb.off()
+
+    The four input and output channels are not provided by this class. Instead,
+    you can create GPIO Zero devices using these pins::
+
+        from gpiozero import Pibrella, LED, Button
+
+        pb = Pibrella()
+        btn = Button(pb.inputs.a)
+        led = LED(pb.outputs.e)
+
+        btn.when_pressed = led.on
+
+    :param bool pwm:
+        If :data:`True` (the default), construct :class: PWMLED instances to
+        represent each LED on the board. If :data:`False`, construct regular
+        :class:`LED` instances.
+
+    :type pin_factory: Factory or None
+    :param pin_factory:
+        See :doc:`api_pins` for more information (this is an advanced feature
+        which most users can ignore).
+
+    .. _Pibrella: http://www.pibrella.com/
+
+    .. attribute:: lights
+
+        :class:`TrafficLights` instance representing the three LEDs
+
+        .. attribute:: red, amber, green
+
+            :class:`LED` or :class:`PWMLED` instances representing the red,
+            amber, and green LEDs
+
+    .. attribute:: button
+
+        The red :class:`Button` objects on the Pibrella
+
+    .. attribute:: buzzer
+
+        A :class:`TonalBuzzer` object representing the buzzer
+
+    .. attribute:: inputs
+
+        A namedtuple of the input pin numbers
+
+        .. attribute:: a, b, c, d
+
+    .. attribute:: outputs
+
+        A namedtuple of the output pin numbers
+
+        .. attribute:: e, f, g, h
+    """
+    def __init__(self, pwm=True, pin_factory=None):
```

Comment:
Similar classes default `pwm` to False - why does this one default it to True?

Reply:
Good spot, should be `False`

Reply:
Don't forget to fix this before merging...

Reply:
Thanks - done

Reply:
Need to update the docstring too :stuck_out_tongue_winking_eye:

Reply:
Oh nearly missed that - done